test/verify/check-storage-luks

```diff
@@ -130,6 +130,10 @@ class TestStorage(StorageCase):
         if m.execute("which clevis || true") == "":
             self.skipTest("No clevis/tang here")
 
+        # HACK - clevis in rhel-8 is too old
+        #
+        can_unlock = "clevis luks unlock" in m.execute("clevis --help 2>&1")
```

Comment:
I don't like this too much, TBH. Imagine clevis changes its CLI interface at some point on an image refresh, then this test would silently skip instead of failing and indicating that we need to adjust it. A static image list would be more robust IMHO.